its/src/test/java/com/sonar/it/scanner/msbuild/sonarqube/ScannerMSBuildTest.java

```diff
@@ -645,7 +644,7 @@ public void testCustomRoslynAnalyzer() throws Exception {
     runBeginBuildAndEndForStandardProject(folderName, "", false, false);
 
     List<Issue> issues = TestUtils.allIssues(ORCHESTRATOR);
-    assertThat(issues).hasSize(2 + 37 + 1);
+    assertThat(issues).hasSize(1 + 37 + 1);
```

Comment:
Pedantic: What does each number come from?

I am guessing C# & VB issues but the third one I don't know.

Reply:
The behavior is not clear from the test. There is an analyzer implemented in the test projects and the following rules are raising issues:
- roslyn.sonaranalyzer.scanner.integrationtest.cs:S1235 x 1
- roslyn.sonaranalyzer.scanner.integrationtest.cs:S1234 x 38
I will merge this PR as is and create another one to improve the test.